app/src/main/java/com/tpp/theperiodpurse/AppScreen.kt

```diff
@@ -248,6 +249,7 @@ fun ScreenApp(
                 modifier = Modifier.fillMaxSize(),
                 contentScale = ContentScale.FillBounds
             )
+            appViewModel.setAllowReminders(hasNotificationsPermissions)
```

Comment:
As far as I can tell, this sets `allowReminders` to be always false instead of fetching from the db, which means the switch in Settings page is always false